capstone/src/main/java/com/google/univiz/api/MapsData.java

```diff
@@ -0,0 +1,4 @@
+package com.google.univiz.api;
+
+/** The MapsData class will contain all college data that is specific to the Maps feature */
+public class MapsData {}
```

Comment:
Could this also be `final` or will be an `AutoValue`?

Reply:
AutoValue :)